src/install.c

```diff
@@ -708,8 +710,10 @@ static gboolean run_bundle_hook(RaucManifest *manifest, gchar* bundledir, const
 		outline = g_data_input_stream_read_line(datainstream, NULL, NULL, NULL);
 		if (outline) {
 			g_clear_pointer(&hookreturnmsg, g_free);
-			hookreturnmsg = outline;
+			hookreturnmsg = g_strdup(outline);
 		}
+
+		g_free(outline);
```

Comment:
What does in comparison to the original code the combination of `g_strdup(outline)` in line 713 and `g_free(outline)` in line 716 change other than adding an additional copy operation?

Reply:
You're right. I've first 'fixed' the gfree and then noticed we need to keep at least the latest content of outline. But did not notice this is handled by the freeing `hookreturnmsg` already.